### PR TITLE
Release 11.1.5 master

### PR DESCRIPTION
**Pull request by kkelly51**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -300,6 +301,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     `example/cumulus-tf/sync_granule_workflow.asl.json` to include `ACL`
     parameter.
 
+## [v11.1.5] 2022-08-10 [BACKPORT]
+**Please note** changes in 11.1.4 may not yet be released in future versions, as
+this is a backport and patch release on the 11.1.x series of releases. Updates that
+are included in the future will have a corresponding CHANGELOG entry in future
+releases.
+
```
Comment (npauzenga) [inline CHANGELOG.md:309]: I think this is missing the actual changelog entry? https://github.com/nasa/cumulus/pull/3077/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15-R22

**Review by npauzenga — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kkelly51) [reply CHANGELOG.md:309]: oooohhhhhhh my b

Comment (kkelly51) [reply CHANGELOG.md:309]: there we go

**Review by npauzenga — APPROVED** [review]
TGTM

**Pull request merged**